universe/IDAllocator.cpp

```diff
@@ -305,7 +305,7 @@ void IDAllocator::SerializeForEmpire(Archive& ar, const unsigned int version, co
                 & BOOST_SERIALIZATION_NVP(m_offset_to_empire_id);
         } else {
             auto temp_empire_id = empire_id;
-            ar  & BOOST_SERIALIZATION_NVP(temp_empire_id);
+            ar  & boost::serialization::make_nvp("m_empire_id", temp_empire_id);
```

Comment:
Might change `"m_empire_id"` to `BOOST_PP_STRINGIZE(m_empire_id)` (and in following cases)

Reply:
Fixed.